pkg/pushsync/pushsync.go

```diff
@@ -211,8 +307,15 @@ func (ps *PushSync) pushToClosest(ctx context.Context, ch swarm.Chunk) (rr *pb.R
 
 		defersFn()
 
-		// find next closest peer
-		peer, err := ps.peerSuggester.ClosestPeer(ch.Address(), skipPeers...)
+		deferFuncs = append(deferFuncs, func() {
```

Comment:
this is already done in the code block below `ClosestPeer` call and is unnecessary. Having it here means that we will also unnecessarily print the error twice in case there was an error in the last iteration _and_ and error from `ClosestPeer`